Storage/StorageTests/Tools/StorageTypeExtensionsTests.swift

```diff
@@ -66,6 +66,30 @@ final class StorageTypeExtensionsTests: XCTestCase {
         XCTAssertEqual(site, storedSite)
     }
 
+    func test_loadOrders_list_by_siteID_and_orderIDs() {
```

Comment:
Suggestion: We could update this test by adding an order with a different site ID other than `sampleSiteID` to validate that only orders from the given `siteID` are loaded.

Reply:
Updated the test in a27533c7133933d91e0cf80da4a3e24259c73ddd.